src/backend/distributed/executor/multi_server_executor.c

```diff
@@ -58,6 +59,13 @@ JobExecutorType(MultiPlan *multiPlan)
 		return MULTI_EXECUTOR_ROUTER;
 	}
 
+	/* if it is not a router executable plan, inform user according to the log level */
+	if (MultiTaskQueryLogLevel != MULTI_TASK_QUERY_INFO_OFF)
+	{
+		ereport(MultiTaskQueryLogLevel, (errmsg("this query is multi-task query, that "
```

Comment:
There's a missing 'a' here, but I'd rephrase a bit more aggressively.  How about something roughly like`errmsg("multi-task about to be executed"), errhint("Queries use multiple tasks if they have to be split into several queries on the workers.")`